ldap/servers/slapd/back-ldbm/cache.c

```diff
@@ -514,6 +515,30 @@ flush_remove_entry(struct timespec *entry_time, struct timespec *start_time)
     }
 }
 
+static inline void
+dbgec_test_if_entry_pointer_is_valid(void *e, void *prev, int slot, int line)
+{
+    /* Check if the entry pointer is rightly aligned and crash loudly otherwise */
+    if ( ((uint64_t)e) & ((sizeof(long))-1) ) {
```

Comment:
Reading this I was wondering if it was valid in little/big endian. I think it is. Am I correct ?